LEGO1/tgl/d3drm/impl.h

```diff
@@ -818,4 +820,6 @@ inline D3DRMLIGHTTYPE Translate(LightType tglLightType)
 // GLOBAL: LEGO1 0x100dd1e0
 // IID_IDirect3DRMMeshBuilder
 
-} /* namespace TglImpl */
+} /* namespace TglImpl *
```

Comment:
```suggestion
} /* namespace TglImpl */
```

Reply:
Oops, I did the edit without review. My bad